rules/integrations/aws/defense_evasion_cloudtrail_logging_deleted.toml

```
@@ -86,9 +88,26 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:cloudtrail.amazonaws.com and event.action:DeleteTrail and event.outcome:success
+event.dataset:aws.cloudtrail
+    and event.provider:cloudtrail.amazonaws.com
+    and event.action:DeleteTrail
+    and event.outcome:success
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "event.action",
+    "event.outcome",
+    "cloud.region",
+    "user_agent.original",
+    "user.name",
+    "related.user",
+    "source.address",
+    "aws.cloudtrail.request_parameters",
+    "aws.cloudtrail.flattened.request_parameters.name",
```

```suggestion
    "aws.cloudtrail.flattened.request_parameters.name",
```
these are both the same for this event so I'd just choose one of them

---

"Add to timeline" is based on key:value pairs which is one of the few purposes for highlighted fields. If `request_parameters` is added for searching, the related docs would have to have the same exact value. If it were the specific `name` key, then that value may be different. So keeping both allows the user to 1) have the context of the API quest in the flyout, two add `name` to timeline and see potentially different actions being taken.

---

Gotcha 👍🏾